server/zanata-frontend/src/app/containers/ProjectVersion/PeoplePage.js

```diff
@@ -11,6 +11,9 @@ class PeoplePage extends Component {
       /* eslint-disable max-len */
         <div className='flexTab wideView'>
           <h2>People</h2>
+          <React.Fragment>
```

Comment:
@kathryngo Can we simply get rid of this one? (keeping the Content inside the div)